cli/flox/doc/flox-search.md

```diff
@@ -7,35 +7,53 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-search - search for packages to install.
+flox-search - search for packages.
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] search `<name>` [ \--refresh ]
+```
+flox [ <general options> ] search
```

Comment:
The ticket was updated to include removing `--refresh` from the command and `<search-term>` description also needs to be updated in `flox search --help`. I think the man page could also describe a bit more how `<search-term>` works (partial match against `relPath`, can contain `@` and a semver specifier)